README.md

```diff
@@ -104,6 +104,27 @@ swaggen generate http://myapi.com/spec --template Templates/Swift  --destination
 
 For the Swift template, a handy option is `name`, which changes the name of the generated framework from the default of `API`. This can be set in the template or by passing in `--option name:MyCoolAPI`.
 
+
+### Option
```

Comment:
Can we call this `Swift Template Options`

Reply:
Ok